treasury/treasury/app.py

```diff
@@ -0,0 +1,248 @@
+        State('small-prices', 'data'),
+        prevent_initial_call=True)(download_small_prices)
+
+    app.callback(
+        Output('download-full-dataframe', 'data'),
+        Input('download-button-full', 'n_clicks'),
+        State('full-sims', 'data'),
+        prevent_initial_call=True)(download_full)
+
+    app.callback(
+        Output('download-small-dataframe', 'data'),
+        Input('download-button-small', 'n_clicks'),
+        State('small-sims', 'data'),
+        prevent_initial_call=True)(download_small)
+
+    app.callback(
+        Output('summary-table', 'children'),
+        Input('lookback-prices', 'data'),
+        Input('ref-ticker', 'value'))(update_summary)
+
+    app.callback(
+        Output('ref-prices', 'data'),
+        Output('lookback-prices', 'data'),
+        Input('start-date', 'value'),
+        Input('end-date', 'value'),
+        State('ref-prices', 'data'),
+        State('lookback-prices', 'data'))(get_update_prices(price_data_loc))
+
+    app.callback(
+        Output('forecast-graph', 'figure'),
+        Output('full-sims', 'data'),
+        Output('small-sims', 'data'),
+        Output('full-prices', 'data'),
+        Output('small-prices', 'data'),
+        Input('lookback-prices', 'data'),
+        Input('ref-ticker', 'value'),
+        Input('forecast-days', 'value'),
+        Input('num-sims', 'value'),
+        Input('trend-scale', 'value'),
+        Input('vol-scale', 'value'),
+        Input('risk-threshold', 'value'),
+        State('forecast-graph', 'figure'),
+        State('asset-values', 'data'))(update_forecast_chart)
+
+    app.callback(
+        Output('price-graph', 'figure'),
+        Input('lookback-prices', 'data'),
+        State('price-graph', 'figure'))(update_price_chart)
+
+    return app
+
+
+def main():
+    parser = argparse.ArgumentParser(description='webapp that processes data files and renders fork information')
+    # parser.add_argument('--resources', help='directory containing resources', required=True)
+    parser.add_argument('--host', help='host ip, defaults to localhost', default='127.0.0.1')
+    parser.add_argument('--port', type=int, help='port for webserver', default=8080)
+    parser.add_argument('--proxy', help='proxy spec of the form ip:port::gateway to render urls', default=None)
+    parser.add_argument('--base_path', help='extension if server is not at root of url', default=None)
+    parser.add_argument('--serve', action='store_true', help='flag to indicate whether server will recieve external requests')
+    parser.add_argument('--price_data_loc', help='path to flat file storing collected data', default='../data/price_data.csv')
+    parser.add_argument('--accounts_loc', help='path to csv with account information', default='../data/accounts.csv')
+    parser.add_argument('--start_date', help='default start date', default='2021-12-01')
+    parser.add_argument('--end_date', help='default end date', default=None)
```

Comment:
start date and end date don't seem to be getting used?

Reply:
They are. These are for the backtest. You'll find it fixed now (wasn't flowing through to the app).